### PR TITLE
meta(icons): Added "liquid" tag to Droplet icon

### PR DESCRIPTION
**Pull request by vBohu**
Bug fix

### CODE REVIEW

**Review by jguddas — APPROVED** [review]

**Comment by karsa-mistmere** [comment]
@vBohu: Thanks, there's definitely much room for improvement concerning the current tags. I've tried to update your PR, but you haven't provided access to your source branch. Can you also add these tags to both `droplet` and `droplets`?

```tsx
    "liquid",
    "fluid",
    "wet",
    "moisture",
    "damp",
    "bead",
    "globule"
```

File: icons/droplet.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
   ],
   "tags": [
     "water",
-    "weather"
+    "weather",
+    "liquid"
```
Comment (jguddas) [inline icons/droplet.json:12]:
```suggestion
    "liquid",
    "fluid",
    "wet",
    "moisture",
    "damp",
    "bead",
    "globule"
```

Comment (karsa-mistmere) [reply icons/droplet.json]: Can you please also add these to `droplets.json`?

**Review by ericfennis — DISMISSED** [review]

**Review by karsa-mistmere — CHANGES_REQUESTED** [review]
Can you please also add these to `droplets.json`?

**Pull request merged**